lib/ui/widget/items/items_view.dart

```diff
@@ -25,26 +25,32 @@ class ItemsView extends StatelessWidget {
                   const SizedBox(
                     height: 20.0,
                   ),
-                  Row(
-                    mainAxisAlignment: MainAxisAlignment.start,
-                    children: [
-                      Expanded(
-                        flex: 2,
-                        child: Text('All Items',
-                            style: GoogleFonts.lato(
-                                textStyle: TextStyle(
-                                    fontWeight: FontWeight.bold,
-                                    fontSize: 14.0,
-                                    color: Colors.grey[800]))),
-                      ),
-                      Expanded(
-                        flex: 1,
-                        child: Row(children: [
-                          const Expanded(
-                              child: Icon(FontAwesome.chevron_right, size: 20)),
-                        ]),
-                      ),
-                    ],
+                  InkWell(
+                    onTap: () {
+                      model.navigateTo(path: Routing.itemsListView);
+                    },
+                    child: Row(
+                      mainAxisAlignment: MainAxisAlignment.start,
+                      children: [
+                        Expanded(
+                          flex: 2,
+                          child: Text('All Items',
+                              style: GoogleFonts.lato(
+                                  textStyle: TextStyle(
```

Comment:
For Text style let's put Theme.of(context).textTheme.headline5.copyWith( 
                          
                          height: 1.67), Let's always put height and use a Theme as we decided so far.